tests/page-objects/dashboard/DashboardPage.ts

```diff
@@ -641,6 +641,14 @@ class DashboardPage extends BasePage {
     async isDescriptionEditMode() {
         await this.description.isEditMode();
     }
+
+    async clickSelectWithTitle(title: string) {
+        await this.page
+            .locator(slct('chartkit-control'))
+            .filter({hasText: title})
+            .filter({has: this.page.locator(slct('chartkit-control-select'))})
```

Comment:
`chartkit-control` & `chartkit-control-select` must be constants, and may be it would better to get it as function arguments (now it looks like hardcode)

Reply:
Updated PO. Added the use of constants. I think it should be a precision selector. Passing `data-qa` arguments will only make it harder to use.